assets/js/translator.js

```diff
@@ -133,8 +133,6 @@ if(modeEnabled('translation')) {
             synchronizeTextareaHeights();
         });
 
-        $(window).resize(synchronizeTextareaHeights);
```

Comment:
Did you figure out why this was added using git blame? I'm concerned about removing it without knowing why.